api-tests/CMakeLists.txt

```diff
@@ -105,6 +105,7 @@ list(APPEND PSA_INITIAL_ATTESTATION_FILES
 # list of supported toolchains
 list(APPEND PSA_TOOLCHAIN_SUPPORT
         GNUARM
+	GNUARM_LINUX
```

Comment:
please add as we discussed compiler_linux in this case  -> GCC_LINUX, for yocto  YOCTO_LINUX